src/site/antora/modules/ROOT/partials/manual/appender-deprecation.adoc

```diff
@@ -0,0 +1,26 @@
+////
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+////
+
+[WARNING]
+====
+*This appender is planned to be removed in the next major release, due to lack of maintainers!*
+Users can either:
+
+* step up and offer to maintain the component. See xref:development.adoc[] for details.
+* maintain a 3.x version as a separate project.
+* switch to another appender.
```

Comment:
```suggestion
*This appender is planned to be removed in the next major release!*
If you are using this library, please get in touch with the Log4j maintainers using link:{logging-services-url}/support.html[the official support channels].
```

I think you can receive the `appender` word as a variable and use this partial everywhere a deprecation notice is needed.